src/App.tsx

```diff
@@ -1,10 +1,24 @@
 import Header from './components/Header/Header';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import HomePage from './pages/HomePage';
```

Comment:
Lige nu er der kun 1 punktum, men plejer det ikke at være 2 for at indikere man går et skridt tilbage i fil stien?
Skal det ikke være ```'../path/to/file'``` ?